mirdata/beatles.py

```diff
@@ -248,7 +254,7 @@ def _load_chords(chords_path):
             chords.append(line[2])
 
     chord_data = utils.ChordData(
-        np.array(start_times), np.array(end_times), np.array(chords)
+        np.array(start_times), np.array(end_times), chords
```

Comment:
while you're at it, want to fix #135 too?

Reply:
yep, monster PR anyway